service/provider/src/main/java/gov/va/vro/service/provider/camel/SlipClaimSubmitRouter.java

```diff
@@ -26,8 +27,8 @@ public class SlipClaimSubmitRouter {
   public String routeClaimSubmit(Object body, @ExchangeProperties Map<String, Object> props) {
     Object diagnosticCodeObj = props.get("diagnosticCode");
     if (diagnosticCodeObj == null) {
-      log.error("No diagnostic code in the body.");
-      return null;
+      log.error(NO_DIAGNOSTIC_CODE_ERROR);
+      throw new CamelProcessingException(NO_DIAGNOSTIC_CODE_ERROR);
```

Comment:
throw exception to avoid an NPE on the controller

Reply:
The compiler is okay with this?

Reply:
Yes, why would it not be?